src/odemis/driver/lakeshore.py

```diff
@@ -108,11 +103,75 @@ def __init__(self, name, role, port, sensor_input='b', output_channel=2, **kwarg
         except LakeshoreError as ex:
             logging.warning("Discarding initial error status: %s", ex)
 
-        # Vigilant attributes of the controller.
-        self.temperature = FloatVA(unit=u"°C", value=self.GetSensorTemperature(), readonly=True)
-        self.targetTemperature = FloatContinuous(value=self.GetSetpoint(), unit=u"°C", range=[-273, 50], setter=self._set_targetTemperature)
-        self.heating = IntEnumerated(value=self.GetHeaterRange(), choices={0: "Off", 1: "Low", 2: "Medium", 3: "High"},
-                                     setter=self._set_heating)
+        # set supported sensor input and output channel values
+        if md == "MODEL335":
+            supported_sensor_inputs = ('A', 'B')
+            supported_output_channels = (1, 2)
+        elif md == "MODEL350":
```

Comment:
Could you add a small comment on line 107 and line 110 explaining MODEL335 and MODEL350?